riverdriver/riverpgxv5/river_pgx_v5_driver.go

```diff
@@ -475,12 +475,14 @@ func (l *Listener) Close(ctx context.Context) error {
 		return nil
 	}
 
-	if err := l.conn.Conn().Close(ctx); err != nil {
-		return err
-	}
```

Comment:
Man, I noticed [this change](https://github.com/riverqueue/river/pull/212/files#diff-445bd06e41eab3a8a89576b7cda31b0e834bbefa4fa969af11c1ce17766b1cceL149) when I reviewed the mega-PR, should have said something 😞

Reply:
Doh. Well, I was going to say "next time", but hopefully nothing like that PR ever happens again :)